core/src/main/kotlin/no/strooped/StroopedController.kt

```diff
@@ -5,33 +5,41 @@ import com.badlogic.gdx.Screen
 import com.badlogic.gdx.utils.ObjectMap
 import no.strooped.service.JoinGameService
 import no.strooped.view.screen.JoinGameScreen
+import no.strooped.view.screen.LobbyScreen
 
-const val TITLE = "Paddle"
+const val TITLE = "Strooped"
 
 class StroopedController : Game() {
     private val screens: ObjectMap<Class<out Screen>, Screen> = ObjectMap()
-
-    lateinit var joinGameService: JoinGameService
+    var username = ""
```

Comment:
Ref the comment bellow about having some sort of singleton. But I also think that you should let username be nullable, so you can make use of the strengths kotlin has with nullable variables